handlers/main.yml

```diff
@@ -1,15 +1,7 @@
 # file: postgresql/handlers/main.yml
 
-  - name: restart postgresql with service
-    service:
-      name: "{{ postgresql_service_name }}"
-      state: restarted
-      enabled: yes
+- name: reload postgresql
```

Comment:
Split out the handlers into separate files, so that many tasks can be done.  I didn't have any luck using `block:` for that.  Also, put any logic for handling different system times in the `restart.yml`, and `reload.yml`... so we don't need multiple different handlers.  Just "restart" and "reaload"